lib/facebook/setup.js

```diff
@@ -84,13 +85,12 @@ module.exports = function fbSetup(api, bot, logError, optionalParser, optionalRe
                 facebookAppSecret: results['Facebook App Secret']
               }
             };
-
             if (!data.variables || (!data.variables.facebookAppSecret && !results['Facebook App Secret']))
               console.log(`\n${color.yellow}Deprecation warning:${color.reset} your bot is not using facebook validation. Please re-run with --configure-fb-bot to set it. This will become mandatory in the next major version. See https://github.com/claudiajs/claudia-bot-builder/blob/master/docs/API.md#message-verification for more information.\n`);
 
             return utils.apiGatewayPromise.createDeploymentPromise(deployment);
           })
-          .then(() => rp.post(`https://graph.facebook.com/v2.6/me/subscribed_apps?access_token=${pageAccessToken}`));
+          .then(() => rp.post(`https://graph.facebook.com/v3.2/${pageID}/subscribed_apps?subscribed_fields=['messages']&access_token=${pageAccessToken}`));
```

Comment:
Is there any other values for `subscribed_fields` that users might use? If yes, maybe we can set a default value to `'messages'` and ask users if they want to provide any additional ones.

Reply:
I believe there is a messages received value that users might be interested in? In case they are creating logs as to how many people click on their messages. The first answer here has all of them:
https://stackoverflow.com/questions/53778224/how-to-solve-the-parameter-subscribed-fields-is-required-error-while-subscribi